tahrir_api/model.py

```diff
@@ -252,12 +258,12 @@ def recipient_default(context):
     Session = sessionmaker(context.engine)()
     person_id = context.current_parameters['person_id']
     person = Session.query(Person).filter_by(id=person_id).one()
-    return unicode(hashlib.sha256(
-        person.email + context.current_parameters['salt']).hexdigest())
+    return str(hashlib.sha256((
+        person.email + context.current_parameters['salt']).encode('utf-8')).hexdigest())
 
 
 def salt_default(context):
-    return unicode(uuid.uuid4())
+    return str(uuid.uuid4())
```

Comment:
if you return `uuid.uuid4().bytes` then you don't need to encode it here https://github.com/fedora-infra/tahrir-api/pull/46/commits/501d0692a8b4cb18d21abee3d147939a9fcdec9b#diff-bbea1d4b7899eebccc87dccb4e388c54R218